pages/showcases/_id.vue

```diff
@@ -23,34 +23,38 @@
 </template>
 
 <script>
+import { mapState } from 'vuex'
 import Progress from '~/components/Progress'
 import CTANext from '~/components/cta/Next'
 import Header from '~/components/Header'
 import Schema404 from '~/components/schema/404'
 import SchemaNutshell from '~/components/schema/Nutshell'
 import page from '../page'
+
 export default {
   components: {
     Header,
     Progress,
     CTANext
   },
-  mixins: [
-    page(self => ({
-      title: self.usecase.title,
-      description: self.usecase.text,
-      schema: SchemaNutshell
-    }))
-  ],
+  
   computed: {
-    usecases () {
-      return require('~/assets/usecases.json')
-    },
+    ...mapState([ 'usecases' ]),
+
     usecase () {
       return this.usecases
         .filter(x => x.id === this.$route.params.id)[0]
     },
+
     schema404 () { return Schema404 }
-  }
+  },
+
+  mixins: [
```

Comment:
I like to keep the mixins on the top, methods, computed and hooks can depends on mixins. Again not any guideline so happy to change that if it makes sense